examples/lora/sensor-receive/Makefile

```diff
@@ -0,0 +1,17 @@
+# Makefile for user application
+
+# Specify this directory relative to the current application.
+TOCK_USERLAND_BASE_DIR = ../../../
+
+# Which files to compile.
+CXX_SRCS := $(wildcard *.cc)
+
+# Use the libtock-c Make system
+EXTERN_LIBS := $(TOCK_USERLAND_BASE_DIR)/libradio/RadioLib
+include $(TOCK_USERLAND_BASE_DIR)/libradio/Makefile
```

Comment:
Should this be `EXTERN_LIBS += ` inside of the libradio Makefile?

[or, looking at the libradio makefile, I don't think this should be needed at all? `EXTERN_LIBS` is for _pre-built_ external libraries (e.g. libc), not stuff that may need building with the app (e.g. b/c of a submodule update)]

Reply:
Yep, it doesn't seem to be required at all. Removed